cypress/e2e/signIn.cy.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types="cypress" />
+describe('Sign in page', () => {
+  beforeEach(()=>{
+    cy.visit('https://the-internet.herokuapp.com/login');
+  });
+
+  it('User should log in with valid credentials', () => {  
```

Comment:
```suggestion
  it('should log in with valid credentials', () => {  
```
each `it` name should continue your describe name